src/sentry/sentry_metrics/use_case_id_registry.py

```diff
@@ -17,6 +17,11 @@ class UseCaseID(Enum):
     UseCaseID.TRANSACTIONS: UseCaseKey.PERFORMANCE,
 }
 
+METRIC_ID_MAPPING: Mapping[UseCaseKey, UseCaseID] = {
```

Comment:
Let's add a TODO here to note that this is temporary and eventually UseCaseKey will have a 1:N relationship with UseCaseID